Documentation/ceph-monitoring.md

```diff
@@ -104,7 +104,7 @@ kubectl create -f deploy/examples/monitoring/rbac.yaml
 ```
 
 2. Make following changes to your CephCluster object (e.g., `cluster.yaml`).
-
+3. Optional: prometheusRule field for overwriting values of existing alerts (Only if monitoring.enabled: true)
```

Comment:
Enabling the alerts is just part of step 2. How about if we just remove this line and leave enabling the rules as comments below?

Reply:
Sounds good :)
Removed.